com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/ClearLightLists.compute

```diff
@@ -1,7 +1,10 @@
 #pragma kernel ClearList
 
 RWStructuredBuffer<uint>  _LightListToClear;
-int _LightListEntries;
+int2 _LightListEntriesAndOffset;
+
+#define _LightListEntries (uint)_LightListEntriesAndOffset.x
+#define _LightListOffset  (uint)_LightListEntriesAndOffset.y
 
 [numthreads(64, 1, 1)]
```

Comment:
you do'nt used _LightListOffset   in the code, don't think it is expected

Reply:
Oh shoot, forgot to stage the full file! Staging now and pushing